src/components/CreateResetPassword/components/SuccessPage/SuccessPageCtrl.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import {
+  withRouter,
+  Redirect,
+} from 'react-router-dom';
+
+import SuccessPage from './SuccessPage';
+
+class SuccessPageCtrl extends Component {
```

Comment:
Why do we need this component? Could we just return redirect component on click event in presentational component?